### PR TITLE
Bug 1673828 - Efficiently remove blank signatures

### PR DESCRIPTION
**Pull request by beatrice-acasandrei**

### CODE REVIEW

File: treeherder/model/management/commands/cycle_data.py [hunk 1]
```diff
@@ -101,6 +101,11 @@ def __init__(
         self.max_runtime = max_runtime or PerfherderCycler.DEFAULT_MAX_RUNTIME
         self.strategies = strategies or RemovalStrategy.fabricate_all_strategies(chunk_size)
 
+    @property
+    def max_timestamp(self):
```
Comment (ionutgoldan) [inline treeherder/model/management/commands/cycle_data.py:105]: `max_timestamp` is ambiguous. We need to rename this property to something more specific.

**Review by ionutgoldan — APPROVED** [review]

**Comment by sarah-clements** [comment]
Sorry for the delay in reviewing. This looks ok to me. @beatrice-acasandrei can you confirm this change has been tested on the prototype2 deployment?

**Comment by beatrice-acasandrei** [comment]
> Sorry for the delay in reviewing. This looks ok to me. @beatrice-acasandrei can you confirm this change has been tested on the prototype2 deployment?

@sarah-clements I don't think that this change was tested on the prototype2 deployment. Ionut Goldan can help with this but unfortunately he is on medical leave at the moment.

**Comment by sarah-clements** [comment]
> > Sorry for the delay in reviewing. This looks ok to me. @beatrice-acasandrei can you confirm this change has been tested on the prototype2 deployment?
> 
> @sarah-clements I don't think that this change was tested on the prototype2 deployment. @ionutgoldan can help with this but unfortunately he is on medical leave at the moment.

No problem, I can help you. I've pushed your branch upstream and deployed it to prototype2. We can let it stay here for a day or so and then I'll set it back to master. I'll check the papertrail logs to make sure there aren't any errors. While Ionut is out, you can ask myself or :camd in the treeherder channel to deploy any of your branches to prototype2 for testing.

**Comment by beatrice-acasandrei** [comment]
> > > Sorry for the delay in reviewing. This looks ok to me. @beatrice-acasandrei can you confirm this change has been tested on the prototype2 deployment?
> > 
> > 
> > @sarah-clements I don't think that this change was tested on the prototype2 deployment. @ionutgoldan can help with this but unfortunately he is on medical leave at the moment.
> 
> No problem, I can help you. I've pushed your branch upstream and deployed it to prototype2. We can let it stay here for a day or so and then I'll set it back to master. I'll check the papertrail logs to make sure there aren't any errors. While Ionut is out, you can ask myself or :camd in the treeherder channel to deploy any of your branches to prototype2 for testing.

@sarah-clements do you have any update regarding the papertrail logs? Thank you for helping with this!

**Comment by sarah-clements** [comment]
> > > > Sorry for the delay in reviewing. This looks ok to me. @beatrice-acasandrei can you confirm this change has been tested on the prototype2 deployment?
> > > 
> > > 
> > > @sarah-clements I don't think that this change was tested on the prototype2 deployment. @ionutgoldan can help with this but unfortunately he is on medical leave at the moment.
> > 
> > 
> > No problem, I can help you. I've pushed your branch upstream and deployed it to prototype2. We can let it stay here for a day or so and then I'll set it back to master. I'll check the papertrail logs to make sure there aren't any errors. While Ionut is out, you can ask myself or :camd in the treeherder channel to deploy any of your branches to prototype2 for testing.
> 
> @sarah-clements do you have any updates regarding the papertrail logs? Thank you for helping with this!

Thanks for the reminder. I don't see any errors pertaining to this in the logs.

**Review by sarah-clements — APPROVED** [review]

**Pull request merged**